install/kubernetes/cilium/Chart.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: v2
 name: cilium
 displayName: Cilium
 home: https://cilium.io/
-version: 1.15.0-dev
-appVersion: 1.15.0-dev
-kubeVersion: ">= 1.16.0-0"
+version: 1.16.0-dev
+appVersion: 1.16.0-dev
+kubeVersion: ">= 1.26.0-0"
```

Comment:
Was this last part of the change on purpose? This has made our CI fail for all clusters with a version of Kubernetes 1.25 or lower (specific to our cloud providers)

Reply:
It got fixed here already: https://github.com/cilium/cilium/pull/29888/files, but my question still stands. Was this on purpose? I would have expected it to be called out in the PR description if that was the case.

Reply:
This was expected to be a pure documentation update to suggest to users that they use newer versions of k8s, which would match the documentation at https://docs.cilium.io/en/v1.15.0-rc.0/network/kubernetes/compatibility/#k8scompatibility . The intent was that this was more of a clerical step rather than a change in policy or explicit decision.

As it turns out, this didn't just suggest that users run newer versions, but *required* that users run newer versions. As such, I completely agree with the revert of this specific change that came through in #29888.